lib/danger/scm_source/git_repo.rb

```diff
@@ -19,6 +19,34 @@ def diff_for_folder(folder, from: "master", to: "HEAD")
       self.log = repo.log.between(from, to)
     end
 
+    def renamed_files
+      # Get raw diff with --find-renames --diff-filter
+      diff = exec(
+        "diff #{self.diff.from} #{self.diff.to} --find-renames --diff-filter=R"
```

Comment:
This adds another dependency for this library (one that may not be on all machines).
I'd *really* like to see this implemented using `ruby-git`s `diff.patch` functionality instead, as that ensures that there aren't any other dependencies required.